scripts/pull_github_pr.sh

```diff
@@ -57,11 +66,6 @@ check_jenkins_job_status() {
   # Fetch Jenkins job last build status without passing credentials explicitly
   lastCompletedBuild="${jenkins_url}/job/${jenkins_job}/lastCompletedBuild/api/json?tree=result"
   lastCompleted=$(curl -s --user $JENKINS_USERNAME:$JENKINS_API_TOKEN $lastCompletedBuild | jq -r '.result')
-  # Handle the case where authentication fails
-  if [[ "$lastCompleted" == *"Authentication required"* ]]; then
-      echo "Failed to authenticate with Jenkins. Please check your session or credentials."
```

Comment:
Why is this no longer needed? After all, you can also set your environment variables incorrectly!
But please replace the phrase "Please check your session or credentials" (what does this even mean - what's a "session"?) to please check your JENKINS_USERNAME and JENKINS_API_TOKEN setting.